.github/ISSUE_TEMPLATE/crate-build-failure.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: Crate Build Failure
+description: My crate failed to build in docs.rs.
+labels: build-failure
+body:
+  - type: markdown
+    attributes:
+      value: |
+        If you need a system dependency added for your crate to build, consider making a PR to [rust-lang/crates-build-env](https://github.com/rust-lang/crates-build-env) instead of opening an issue here. There are detailed instructions for this [on this page](https://forge.rust-lang.org/docs-rs/add-dependencies.html).
+  - type: input
+    attributes:
+      label: Crate name
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: build failure link
```

Comment:
There are other labels that don't start with a capitalized letter. Once all labels are capitalized we can merge this!

Reply:
Ah, didn't know you meant all of them 😄 

I'll update it later